learner.rkt

```diff
@@ -13,6 +13,8 @@
     (error "ext2-∇ is not provided by the learner implementation")))
 
 (provide
+ tolerance
+
  len ref refr
```

Comment:
C2: Break out all these fixes to learner and other implementations into their own separate PR.